tests/examples-projects/nodejs-version-shell/shell.nix

```diff
@@ -0,0 +1,9 @@
+{ pkgs ? import ../../../nix {} }:
+let
+  node = pkgs.nodejs-10_x;
+in
+assert pkgs.nodejs == node -> throw "default nodejs version has been updated. Please update the test";
```

Comment:
We should probably leave a line here what the expectation is. In this case we do want to ensure that `nodejs` IS NOT the same as `nodejs-10_x`.

Reply:
Good thinking, the rationale isn't immediately clear